output/test.js

```diff
@@ -1,6 +1,6 @@
-var getIt = require('../');
+var index = require('../index.js');
```

Comment:
I'll take care of all of these issues right now and keep them on the same pull request. Is that a good way to handle it?